server/src/core/event/event.entity.ts

```diff
@@ -0,0 +1,9 @@
+import { ObjectType, Field, Int } from '@nestjs/graphql';
+
+@ObjectType()
+export class Event {
+  @Field(() => Int, {
```

Comment:
@FelixMalfait `Boolean` instead of `Int` should be used, no need to import `Boolean`